TESTS/mbed_drivers/lp_ticker/main.cpp

```diff
@@ -142,7 +142,7 @@ void test_detach(void)
 {
     LowPowerTicker ticker;
     bool ret;
-    const float ticker_time_s = 0.1f;
+    const s_timestamp_t ticker_time_s = 1;
```

Comment:
If this is now one second instead of 0.1s then you should increase wait_time_ms

Reply:
The wait tiume has been incremented by the same factor. d435b1d